src/CoreBundle/Resources/public/javascript/cms.js

```diff
@@ -5,13 +5,30 @@ CHAMELEON.CORE = CHAMELEON.CORE || {};
 
 var sLastDialogID = null;
 
+CHAMELEON.CORE.processingDialog = function (state) {
```

Comment:
Mostly good now, but still I think this should be addressed:

"This is not a dialog, as there is no user interaction, but a modal, so something like "modal" could be used instead of "dialog"."

Reply:
renamed to show/hideProcessingModal (methods and DIV container)